meterfs/test_meterfs.yaml

```
@@ -0,0 +1,16 @@
+test:
+    type: unit
+    targets:
+        include: [host-pc]
```

```suggestion
    targets:
        value: [host-pc]
```
If `value` is not given then runner executes tests on default targets which are `[ia32-generic]`. So as a result tests are executed on targets [`ia32-generic`, `host-pc`]. Use `value` instead of `include` -- it will overwrite the default targets and tests will be run only on `host-pc`.

---

Done.